samples/electron-app/src/main/main.ts

```diff
@@ -0,0 +1,55 @@
+import { app, BrowserWindow } from 'electron';
+import * as path from 'path';
+import { ElectronAuthenticator } from '@microsoft/mgt-electron-provider/dist/Authenticator';
+export default class Main {
+  static application: Electron.App;
+  public static mainWindow: Electron.BrowserWindow;
+  static authProvider: ElectronAuthenticator;
+
+  static main(): void {
+    Main.application = app;
+    Main.application.on('window-all-closed', Main.onWindowAllClosed);
+    Main.application.on('ready', Main.onReady);
+  }
+  static onWindowAllClosed(arg0: string, onWindowAllClosed: any) {
+    Main.application.quit();
+  }
+
+  static onReady() {
+    Main.createMainWindow();
+    Main.authProvider = new ElectronAuthenticator({
+      clientId: 'e5774a5b-d84d-4f30-892e-c4f73a503943',
+      authority: 'https://login.microsoftonline.com/29d87d10-93c7-4330-8d09-d6ac77ae5af2',
```

Comment:
removing authority breaks the provider. I expect authority to be optional and by default to use common authority. 

Also, for this sample, once you've fixed authority, consider commenting out the authority line to make it easier for the sample to run with any account (but keep the line commented out to show that the dev can add authority)